### PR TITLE
Added data_collection_from_aws_s3 file

### PR DESCRIPTION
**Pull request by SindhuParkaviS**
The PowerShell code is to copy the data from AWS S3 using the folder paths, unzips the files, and stores them in the local folder.

### CODE REVIEW

File: Data_Collection_group/Data_Collection_Readme.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## data Collection - Readme.md
```
Comment (professor-greebie) [inline Data_Collection_group/Data_Collection_Readme.md:1]: Please place this in the main folder under README.md so it can be viewed on the front page.

**Review by professor-greebie — CHANGES_REQUESTED** [review]
Please move README.md file to main folder and move .ps1 file to scripts please.

File: Data_Collection_group/Data_Collection_Readme.md (hunk 1 above)
Comment (SindhuParkaviS) [reply Data_Collection_group/Data_Collection_Readme.md]: Hi Professor, the requested changes have been done. Please review and approve the request.

**Review by B055man4 — APPROVED** [review]
LGTM

**Review by professor-greebie — APPROVED** [review]
LGTM

**Pull request merged**